services/harmony/env-defaults

```diff
@@ -398,13 +398,13 @@ HARMONY_REGRIDDER_LIMITS_MEMORY=512Mi
 HARMONY_REGRIDDER_INVOCATION_ARGS='python -m harmony_regridding_service'
 HARMONY_REGRIDDER_SERVICE_QUEUE_URLS='["sds/harmony-regridder:latest,http://localstack:4566/queue/harmony-regridder.fifo"]'
 
-SWOT_REPROJECT_IMAGE=sds/swot-reproject:latest
-SWOT_REPROJECT_REQUESTS_CPU=128m
-SWOT_REPROJECT_REQUESTS_MEMORY=128Mi
-SWOT_REPROJECT_LIMITS_CPU=128m
-SWOT_REPROJECT_LIMITS_MEMORY=512Mi
-SWOT_REPROJECT_INVOCATION_ARGS='python swotrepr.py'
-SWOT_REPROJECT_SERVICE_QUEUE_URLS='["sds/swot-reproject:latest,http://localstack:4566/queue/swot-reproject.fifo"]'
+SWATH_PROJECTOR_IMAGE=ghcr.io/nasa/harmony-swath-projector:latest
+SWATH_PROJECTOR_REQUESTS_CPU=128m
+SWATH_PROJECTOR_REQUESTS_MEMORY=128Mi
+SWATH_PROJECTOR_LIMITS_CPU=128m
+SWATH_PROJECTOR_LIMITS_MEMORY=512Mi
+SWATH_PROJECTOR_INVOCATION_ARGS='python -m swath_projector'
```

Comment:
The Swath Projector repository (and Docker image) was restructures in [this commit](https://github.com/nasa/harmony-swath-projector/commit/da7b2b56c43408fee3a7b48fb2de7d6f3a1c73ff). This now aligns with the [guidelines](https://wiki.earthdata.nasa.gov/display/HARMONY/Harmony+Services+General+Repository+Structure) for a backend service repository. The important thing is it's all now in a `swath_projector` subdirectory, which has a `__main__.py` that is now executed when the service is called.